AndorsTrail/res/layout/heroinfo_inventory.xml

```diff
@@ -3,40 +3,86 @@
 	xmlns:android="http://schemas.android.com/apk/res/android"
 	android:layout_width="match_parent"
 	android:layout_height="match_parent"
-	android:orientation="vertical"
```

Comment:
I corrected the file online, since (weirdly) my local version showed "vertical" and also a "left" tag that wasn't online.